src/units.jl

```diff
@@ -302,6 +302,6 @@ factory defaults, this function will return a product of powers of base SI units
 (as [`Unitful.FreeUnits`](@ref)).
 """
 @generated function upreferred(x::Dimensions{D}) where {D}
-    u = *(FreeUnits{((Unitful.promotion[name(z)]^z.power for z in D)...,),()}())
+    u = prod((Unitful.NoUnits, (Unitful.promotion[name(z)]^z.power for z in D)...))
```

Comment:
```suggestion
    u = prod((NoUnits, (promotion[name(z)]^z.power for z in D)...))
```
This is part of the `Unitful` module, so these module qualifiers aren’t necessary.